mangOH-yellow-Manifest.json

```diff
@@ -4,25 +4,25 @@
     "info": {
         "name": "mangOH-%MANGOH_BOARD%-%TARGET%",
         "version": "%VERSION%",
-        "description": "SDK for mangOH %MANGOH_BOARD% with %TARGET%",
+        "description": "SDK for mangOH %MANGOH_BOARD% with %TARGET%, based on Legato version %LEGATO_VERSION% and %TARGET% release %MODEM_RELEASE_VERSION%",
         "master": true,
         "requires": [
             "swi-license_1.1",
```

Comment:
is this supposed to be 1.2 now?

Reply:
Yeah, I fixed that in a later commit.  You're looking at an outdated version of the code.